### PR TITLE
Externally Populated Spin Deck and Gate URL's for Saas

### PR DESCRIPTION
**Pull request by lakkireddys-opsmx**

### CODE REVIEW

File: charts/oes/values.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ installSpinnaker: true
 ## Installation mode
 ## Available installation modes OES-AP, OES, AP, None
 ##
-installationMode: OES-AP
+installationMode: OES
```
Comment (abhinayopsmx) [inline charts/oes/values.yaml]: Why did we change the default installation mode?

**Comment by lakkireddys-opsmx** [comment]
Externally Populated Spin Deck and Gate URL's for Saas

Please review the changes made.

File: charts/oes/values.yaml [hunk 2]
```diff
@@ -864,6 +864,13 @@ spinnaker:
 
   ## Centralized Monitoring enable for Prometheus
   enableCentralMonitoring: false
+ 
+  ## Populate your Spinnaker Deck and Gate URL's in case if you are using SaaS 
+  security:
+    enabled: false
+    saas:
+      apioverrideBaseUrl: /ofc
```
Comment (abhinayopsmx) [inline charts/oes/values.yaml:872]: These urls should be indicative of how actual urls should be. and we should make helm chart behave in such a way that same urls are used in ingress resources that needs to be created.

**Review by abhinayopsmx — CHANGES_REQUESTED** [review]
Please think in user's perspective on how we can fully automate the process. User should just give the necessary urls in the values.yaml and the same urls should be used everywhere. Helm installation should also create ingress resources automatically by using the urls and annotations specified in the values.yaml

**Comment by abhinaybyrisetty** [comment]
Handled these in OES 3.5 release in an alternative way

**Pull request closed without merging**